ssclj/jar/src/com/sixsq/slipstream/ssclj/middleware/cimi_params_impl.clj

```diff
@@ -103,9 +104,9 @@
   (s/join " and " (map #(str "(" % ")") filters)))
 
 (defn throw-illegal-for-invalid-filter
-  [parse-result]
+  [filter-param parse-result]
   (if (insta/failure? parse-result)
-    (throw (Exception. (str "Invalid filter: " (insta/get-failure parse-result))))
+    (throw (u/ex-bad-CIMI-filter filter-param))
```

Comment:
Don't you lose the actual parsing error by ignoring `parse-result`?  Having that error message is likely to be more useful than citing the standards document.